<launch>
    <arg name="device_id" default="IMU"/>
    <node name="imu" pkg="peripherals" type="imu" respawn="true" respawn_delay="1" >
        <param name="device_id" value="$(arg device_id)"/>
    </node>
</launch>
